source/index.md

```diff
@@ -28,10 +28,12 @@ The following OpenSSL version(s) are FIPS validated:
 
 &nbsp;OpenSSL Version&nbsp; | &nbsp;Certificate&nbsp; | &nbsp;Security Policy&nbsp;
 :-: | :-: | :-:
-3.0.0 | [certificate][cert 3.0.0] | [security policy][secpol 3.0.0]
+3.0.8 | [certificate][cert4282] | [security policy][secpol 3.0.8]
+3.0.0 | [certificate][cert4282] | [security policy][secpol 3.0.0]
 
-[cert 3.0.0]: https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282
-[secpol 3.0.0]: https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp4282.pdf
+[cert4282]: https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282
```

Comment:
Interestingly because this was a 3SUB the 3.0.0 version is no longer mentioned on the CSRC. It is very much debatable whether using 3.0.0 further is considered FIPS compliant.